gtest-parallel

```diff
@@ -58,6 +58,9 @@ class SigintHandler(object):
   def __sigint_handler(self, signal_num, frame):
     with self.__lock:
       self.__on_sigint()
+  def interrupt(self):
```

Comment:
Maybe we could get rid of __sigint_handler and use this method instead?
Is interrupt a good name?